src/ConfigChanges/ConfigFile.js

```diff
@@ -193,8 +193,14 @@ function resolveConfigFilePath (project_dir, platform, file) {
         } else if (file.endsWith('config.xml')) {
             filepath = path.join(project_dir, 'app', 'src', 'main', 'res', 'xml', 'config.xml');
         } else if (file.endsWith('strings.xml')) {
+            // Plugins really shouldn't mess with strings.xml, since it's able to be localized
             filepath = path.join(project_dir, 'app', 'src', 'main', 'res', 'values', 'strings.xml');
         }
```

Comment:
I think eslint will give you an error here if the bracket is not on the same line as the else if...
```} else if (file.match(/res\/xml/)) {```